packages/config/config/devices/0x027a/templates/zooz_template.json

```
@@ -251,5 +251,24 @@
 				"value": 0
 			}
 		]
+	},
+	"paddle_programming": {
+		"label": "Paddle Programming",
+		"description": "Disables inclusion/exclusion commands on paddle for use with scene controls.",
```

Can we merge the description and the label into a somewhat short label? "Paddle Programming" doesn't sound intuitive at first glance. Maybe even invert the meaning like this?
```suggestion
		"label": "Scene Controls",
		"description": "When enabled, inclusion/exclusion via the paddle is disabled.",
```
(with inverted Enable/Disable options)

---

I'm all for finding a clearer way to put it but labelling it scene control would probably be more confusing as these switches already have a setting called scene control.  That other setting configures the switch so that they trigger a scene event any time the paddle is pressed (up to 5x up or down).  This is a separate attribute that just disables the programming shortcuts (3x up, 3x down) which normally override the 3x up/down scene controls.  Without changing attrib 26 scene controls still work for all the button presses except 3x down which will put your switch in exclusion mode.

---

Here's that other param called scene control according to the manufacturer:
```
Enable/Disable Scene Control
Parameter 13: Enable or disable scene control functionality for quick double tap triggers.

Values: 0 – scene control disabled (default). 1 – scene control enabled.

Size: 1 byte dec
```

---

I see - then I have no better idea for the label at the moment.